umap/static/umap/js/umap.icon.js

```diff
@@ -14,11 +17,22 @@ U.Icon = L.DivIcon.extend({
     }
   },
 
+  _setRecent: function (url) {
+    if (L.Util.hasVar(url)) return
+    if (url === this.map.options.default_iconUrl) return
```

Comment:
To change with `U.DEFAULT_ICON_URL` [from here](https://github.com/umap-project/umap/pull/1609/files#diff-c616c7abb3defc7b125bc302e655bde24254137640b762135b0eefc61910ece7R142).

Reply:
Done! :)